### DISCUSSION TITLE
Formato de cron del admin es distinto al que usa el schedule?

### DISCUSSION THREAD

**Discussion by jdurandpaz** [Cotalker/documentation#303] [4. Bug Reports] [answered]
Al crearlo por el admin con la siguiente configuración:

![image](https://user-images.githubusercontent.com/50208672/123490306-c71f3080-d5e1-11eb-8d0c-49c0373f1175.png)

El cron queda de la siguiente forma:

![image](https://user-images.githubusercontent.com/50208672/123490351-dc945a80-d5e1-11eb-9616-ce586edab08d.png)

Revisando otros crones antiguos, la notación es distinta ya que considera *seg - min -hora - dia -mes - dia semana, por lo que la notación correcta debería ser 0 1 0 * * * -> 12:01 am todos los dias

**Comment by agallardol** [comment, marked as answer]
@jdurandpaz

**Técnicamente**

Soportamos cron con granularidad de seis campos:

Seconds: 0-59
Minutes: 0-59
Hours: 0-23
Day of Month: 1-31
Months: 0-11 (Jan-Dec)
Day of Week: 0-6 (Sun-Sat)

**Admin**

En la interfaz del admin permitimos construir cron jobs solo con 5 campos

Minutes: 0-59
Hours: 0-23
Day of Month: 1-31
Months: 0-11 (Jan-Dec)
Day of Week: 0-6 (Sun-Sat)

Una herramienta que ayuda para construir o verificar estas expresiones es [Crontab](https://crontab.guru/) (solo soporta 5 dígitos), al poner las expresiones que comentaste notamos que representan lo mismo, solo que una intenta usar el sexto dígito (Crontab no lo soporta)

![Screen Shot 2021-07-05 at 09 48 29](https://user-images.githubusercontent.com/4631063/124481418-3f21ef00-dd76-11eb-96f4-24193fa10d95.png)

![Screen Shot 2021-07-05 at 09 48 37](https://user-images.githubusercontent.com/4631063/124481434-447f3980-dd76-11eb-8a46-13c29098c50a.png)
